ProductionModePlugin.js

```diff
@@ -1,5 +1,5 @@
 var CommonJsRequireDependency = require("webpack/lib/dependencies/CommonJsRequireDependency");
-var ConcatSource = require("webpack/lib/ConcatSource");
+var ConcatSource = require("webpack-sources").ConcatSource;
```

Comment:
If using `require("webpack-sources")`, this needs to be declared as a `dependency` in `package.json`.